draft-ietf-httpbis-digest-headers.md

```diff
@@ -1296,6 +1314,28 @@ Status:  permanent
 
 Specification document(s):  {{digest}} of this document
 
+## Want-Content-Digest Field Registration
```

Comment:
I think this is ok for the time being. I'll follow up with an editorial suggestion to the IANA section once we land these changes to main branch.